README.md

```diff
@@ -41,7 +41,7 @@ Please *do not fork or submit pull requests* to this repository.
 
 Please timebox your solution to one day total.
 
-* Implement your solution using JavaScript or Python.
+* Implement your solution using JavaScript or Python/Java.
```

Comment:
We don't want Java on ours, so let's not be so specific. How about 
`Implement your solution using the language agreed upon with the hiring manager`